dist/style.css

```diff
@@ -298,56 +312,151 @@ html.inboxy.dark-theme .date-row {
 /** 
  * Archive button 
  */
+html.inboxy .bundle-row .archive-bundle,
+html.inboxy .date-row .archive-bundle {
+    background-image: url('chrome-extension://__MSG_@@extension_id__/assets/check-all.svg');
+}
 html.inboxy .brq,
 html.inboxy .ar8 {
     background-image: url('chrome-extension://__MSG_@@extension_id__/assets/check.svg');
 }
 
+html.inboxy.dark-theme .bundle-row .archive-bundle,
+html.inboxy.dark-theme .date-row .archive-bundle {
+    background-image: url('chrome-extension://__MSG_@@extension_id__/assets/light-check-all.svg');
+}
 html.inboxy.messages-dark-theme .brq,
 html.inboxy.dark-theme .ar8 {
     background-image: url('chrome-extension://__MSG_@@extension_id__/assets/light-check.svg');
 }
 
 @-moz-document url-prefix() {
+    html.inboxy .bundle-row .archive-bundle,
+    html.inboxy .date-row .archive-bundle {
+        background-image: url('moz-extension://__MSG_@@extension_id__/assets/check-all.svg');
+    }
     html.inboxy .brq,
     html.inboxy .ar8 {
         background-image: url('moz-extension://__MSG_@@extension_id__/assets/check.svg');
     }
 
+    html.inboxy.dark-theme .bundle-row .archive-bundle,
+    html.inboxy.dark-theme .date-row .archive-bundle {
+        background-image: url('moz-extension://__MSG_@@extension_id__/assets/light-check-all.svg');
+    }
     html.inboxy.messages-dark-theme .brq,
     html.inboxy.dark-theme .ar8 {
         background-image: url('moz-extension://__MSG_@@extension_id__/assets/light-check.svg');
     }
 }
 
+html.inboxy.hide-archive-action .bundle-row .archive-bundle,
+html.inboxy.hide-archive-action .date-row .archive-bundle,
+html.inboxy.hide-archive-action .brq,
+html.inboxy.hide-archive-action .ar9 {
+    display: none;
+}
+
+/**
+ * Delete button
+ */
+html.inboxy .bundle-row .delete-bundle,
+html.inboxy .date-row .delete-bundle,
+html.inboxy .ar9 {
+    background-image: url('chrome-extension://__MSG_@@extension_id__/assets/trash.svg');
+}
+
+html.inboxy.dark-theme .bundle-row .delete-bundle,
+html.inboxy.dark-theme .date-row .delete-bundle,
+html.inboxy.dark-theme .ar9 {
+    background-image: url('chrome-extension://__MSG_@@extension_id__/assets/light-trash.svg');
+}
+
+@-moz-document url-prefix() {
+    html.inboxy .bundle-row .delete-bundle,
+    html.inboxy .date-row .delete-bundle,
+    html.inboxy .ar9 {
+        background-image: url('moz-extension://__MSG_@@extension_id__/assets/trash.svg');
+    }
+
+    html.inboxy.dark-theme .bundle-row .delete-bundle,
+    html.inboxy.dark-theme .date-row .delete-bundle,
+    html.inboxy.dark-theme .ar9 {
+        background-image: url('moz-extension://__MSG_@@extension_id__/assets/light-trash.svg');
+    }
+}
+
+html.inboxy.hide-delete-action .bundle-row .delete-bundle,
+html.inboxy.hide-delete-action .date-row .delete-bundle,
+html.inboxy.hide-delete-action .ar9 {
+    display: none;
+}
+
+/**
+ * Select button
+ */
+html.inboxy .bundle-row .select-bundle,
+html.inboxy .date-row .select-bundle,
+html.inboxy .select-all {
```

Comment:
I'm not sure if I missed something, but I think the `select-all` class name is redundant with `select-bundle`. It seems to me that you can remove `select-all` entirely throughout this changeset.